NuGet.config

```diff
@@ -7,5 +7,6 @@
   </fallbackPackageFolders>
   <packageSources>
     <clear/>
+    <add key="arcade" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
(nit) the key name here seems odd.  If your intention was to tell readers that this is necessary for arcade, then maybe a comment would be better.

Reply:
That was just a blind copy from arcade. I'll update it.